### PR TITLE
[Feature][Branch] create snackbar component

### PR DESCRIPTION
**Pull request by taejs**
## Description

created Snackbar component(draft)
created test
created story
image

## Related Issues

derived from #46 

## Tests

I added the following tests:
Snackbar.test.tsx (renders without crashing)

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui-native/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: src/components/shared/Snackbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+export interface SnackbarProps {
+  text: string;
+  testId?: string;
+}
+
+function Snackbar(props: SnackbarProps): React.ReactElement {
+  return (
+    <Container
+      testId={props.testId}
```
Comment (hyochan) [inline src/components/shared/Snackbar/index.tsx]: testID

**Review by hyochan — CHANGES_REQUESTED** [review]
I think it'd be good if you can customize the styles for `View` and `Text`.

**Comment by hyochan** [comment]
You are directly pushing it to master and you've labeled `branch`! Is this wanted or should we remove `branch` tag?

**Comment by taejs** [comment]
> You are directly pushing it to master and you've labeled `branch`! Is this wanted or should we remove `branch` tag?

Oh, I made a mistake.
I intended to push this PR to `feat/snackbar` branch
changed the base branch to  `feat/snackbar`

**Comment by taejs** [comment]
> I think it'd be good if you can customize the styles for `View` and `Text`.

That's right. It's a draft of snackbar component.
So it'll be developed soon on a feature branch.

**Comment by hyochan** [comment]
@Ta2Rim Looks great. @godon019 Please give further assistance 😀 .

**Review by hyochan — APPROVED** [review]

**Review by godon019 — APPROVED** [review]
This is great! Thank you @Ta2Rim !

**Comment by JongtaekChoi** [comment]
It's so cool! 
I have rebased this `feat/snackbars` branch to the `master` branch. It caused conflict at `.vscode/settings.json` file. So I added some resolving commit. 
Thank you @Ta2Rim

**Pull request merged**